packages/dds/tree/api-report/tree.api.md

```diff
@@ -25,9 +25,11 @@ import type { TSchema } from '@sinclair/typebox';
 // @internal
 export function adaptEnum<TScope extends string, const TEnum extends Record<string, string>>(factory: SchemaFactory<TScope>, members: TEnum): (<TValue extends TEnum[keyof TEnum]>(value: TValue) => {
     readonly value: TValue;
+    readonly "__#124291@#brand": unknown;
```

Comment:
the TypeScript compiler seems to generate this number somehow as part of its handling of JS private fields. It seems to only show up like this in some edge case which is used in the enum helpers. I wonder if I can refactor them to make it go away.

Reply:
I found a fix.